docs/usage/index.md

```diff
@@ -129,7 +129,33 @@ win-vind uses **Run Commands** style configuration method. If you've ever writte
 #### What is a mapping?
 A typical key binding is a simple one that assigns a function from a set of keys. In contrast, the mapping employed in Vim is a kind of extension of key binding, a mechanism that allows recursive key assignment and command definition.
 
-Mapping is defined using the `map` or `noremap` command. Examples of recursive key assignments are shown below.
+Mapping is defined using a variety of map commands known in vim. The most commonly used are
```

Comment:
- Please revise from _vim_ to _Vim_.
- Since `The most commonly used are` is before the list of items, please add a colon at the end like `The most commonly used are:`.